README.md

```diff
@@ -189,10 +189,17 @@ of the main ORT configuration file. It consists of sections related to different
 of these sections and the properties they can contain is described together with the corresponding sub components.
 
 While the file is rather static, there are means to override configuration options for a specific run of ORT or to
-customize the configuration to a specific environment. The following options are supported:
-
-* You can override the values of properties on the command line using the `-P` option. The option expects a
-  key-value pair. The key must define the full path to the property to be overridden, e.g.
+customize the configuration to a specific environment. The following options are supported whereas the priority is in the
+same order as presented:
+
+* Properties can be overridden with environment variables by labeling a variable with the full property path.
```

Comment:
I think it now becomes strange to use the term "overridden" here, as this is the highest priority which overrides nothing that was declared before. Instead of using "overridden" I just say "Properties can be defined ...". Similar in more places below.

Also, what do you mean by "labeling" a variable? I believe you just mean "naming". So how about:

"Properties can be defined via environment variables by using the full property path as the variable name."